src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1559,6 +1553,16 @@ public String toVariableNameString(EnchantmentOffer eo) {
 			.name("Experience Cooldown Change Reason")
 			.description("Represents a change reason of an <a href='events.html#experience cooldown change event'>experience cooldown change event</a>.")
 			.since("INSERT VERSION"));
+
+		if (Skript.classExists("org.bukkit.entity.EntitySnapshot")) {
+			Classes.registerClass(new ClassInfo<>(EntitySnapshot.class, "entitysnapshot")
+				.user("entity ?snapshots?")
+				.name("Entity Snapshot")
+				.description("Represents an entity snapshot.")
```

Comment:
```suggestion
				.description("Represents a snapshot of all the data of the provided entities. This data includes the values of all the entities' attributes at the time of this expression being called. This can only be used to copy the full attributes of an entity. Individual attributes cannot be modified or retrieved.")
```